src/gui/MainWindow.cc

```diff
@@ -2125,6 +2130,37 @@ void MainWindow::sendToPrintService()
 #endif // ifdef ENABLE_3D_PRINTING
 }
 
+void MainWindow::sendToCmd()
+{
+#ifdef ENABLE_3D_PRINTING
+  QString cmd = QString::fromStdString(Settings::Settings::cmdPrintCmd.value());
+
+  if (cmd.trimmed().isEmpty()) {
+    LOG(message_group::Error, Location::NONE, "", "Print cmd connection not configured. Please check preferences.");
+    return;
+  }
+
+  static QTemporaryFile exportFile{QDir::temp().filePath("OpenSCAD.XXXXXX.stl")};
+  if (!exportFile.open()) {
+    LOG(message_group::Error, Location::NONE, "", "Could not open temporary file.");
+    return;
+  }
+  const QString exportFilename = exportFile.fileName();
+
+  //Render the stl to a temporary file:
+  ExportInfo exportInfo = createExportInfo(FileFormat::STL, exportFilename, activeEditor->filepath);
+  exportFileByName(this->root_geom, exportInfo);
+
+  cmd.replace("#1", exportFilename);
+  LOG(message_group::None, Location::NONE, "", qPrintable(cmd));
+  if (fork() == 0) {
+    // Run cmd in background
+    std::system(qPrintable(cmd));
+    exit(0);
+  }
```

Comment:
This will not work on Windows.

As this is part of the GUI anyway, using `QProcess::startDetached` should work cross platform and allows simple error checking if the command actually was started.

Reply:
updated to use `startDetached()`